ui/src/api/secrets/query/get.ts

```diff
@@ -39,5 +39,9 @@ export const useSecrets = () => {
     }),
     queryFn: fetchSecrets,
     enabled: !!namespace,
+    select: (response) => ({
+      ...response,
+      data: [...response.data].sort((a, b) => a.name.localeCompare(b.name)),
```

Comment:
Not mandatory, but just for FYI: we actually have a `sortByName` function already. So you could do `sort(sortByName)` for the benefit of slightly more readability.